xds/internal/resolver/serviceconfig.go

```diff
@@ -28,7 +28,7 @@ import (
 	"sync/atomic"
 	"time"
 
-	xxhash "github.com/cespare/xxhash/v2"
+	"github.com/cespare/xxhash/v2"
```

Comment:
I believe this import rename is intentional. See https://github.com/grpc/grpc-go/pull/7711#discussion_r1803726840

Reply:
Applies to other places where this is being renamed as well. Thanks.